dotnet/samples/Demos/StepwisePlannerMigration/README.md

```diff
@@ -0,0 +1,130 @@
+# Function Calling Stepwise Planner Migration
+
+This demo application shows how to migrate from FunctionCallingStepwisePlanner to a new recommended approach for planning capability - Auto Function Calling.
+The new approach produces the results more reliably and uses fewer tokens compared to FunctionCallingStepwisePlanner.
+
+## Prerequisites
+
+1. [OpenAI](https://platform.openai.com/docs/introduction) subscription.
+2. Update `appsettings.Development.json` file with your configuration for `OpenAI` section or use .NET [Secret Manager](https://learn.microsoft.com/en-us/aspnet/core/security/app-secrets) (recommended approach):
+
+```powershell
+# OpenAI
+dotnet user-secrets set "OpenAI:ChatModelId" "... your model ..."
```

Comment:
Suggest a model to use here, we need to make it clear that only OpenAI models that support function calling will work

Reply:
Thanks, updated!